openpower/package/alternate-gcc/Config.in

```diff
@@ -0,0 +1,22 @@
+config BR2_PACKAGE_ALTERNATE_GCC
+		bool "alternate-gcc"
+		select BR2_PACKAGE_ALTERNATE_BINUTILS
+		help
+		  Build gcc for an Alternate Toolchain, for use in
+		  certain packages.
+
+
+if BR2_PACKAGE_ALTERNATE_GCC
+
+config BR2_ALTERNATE_GCC_VERSION
+		string "Alternate gcc version"
+		default "6.5.0"
```

Comment:
is 6.5.0 what we've been using? I feel like we've had GCC 8 at some point

Reply:
We were using 6.5.0 from Power8 builds, what the original PR #3564 was trying to solve.. I'll wait for the CI to wrap-up for us to have a baseline and will push a change to get it back to GCC 8.

Reply:
moved to 8.4.0